Trying `make changelog`.

``` console
$ make changelog
packaging/release/changelogs/changelog.py release -vv && packaging/release/changelogs/changelog.py generate -vv
Traceback (most recent call last):
  File "packaging/release/changelogs/changelog.py", line 11, in <module>
    import docutils.utils
ImportError: No module named docutils.utils
Makefile:244 : la recette pour la cible « changelog » a échouée
make: *** [changelog] Erreur 1
```

--- a/changelogs/fragments/always_run_removal.yaml
+++ b/changelogs/fragments/always_run_removal.yaml
@@ -1,2 +1,2 @@
 removed_features:
-- "removed the deprecated always_run task option, please use `check_mode: no` instead"
+- "removed the deprecated always_run task option, please use ``check_mode: no`` instead"

--- a/changelogs/fragments/postgresql_user-encrypted-fixes.yml
+++ b/changelogs/fragments/postgresql_user-encrypted-fixes.yml
@@ -1,7 +1,7 @@
 ---
 minor_changes:
-- "`postgresql_user` module changed `encrypted=yes` to be the default. This
+- "``postgresql_user`` module changed ``encrypted=yes`` to be the default. This
    shouldn't break any current playbooks, the module will just store passwords
    hashed by default. This change was done because Postgres 10 dropped support for
-   `UNENCRYPTED` passwords and because all versions since Postgres 7.2 support
+   ``UNENCRYPTED`` passwords and because all versions since Postgres 7.2 support
    storing encrypted passwords."

--- a/changelogs/fragments/powershell_use_set_tmpdir.yaml
+++ b/changelogs/fragments/powershell_use_set_tmpdir.yaml
@@ -1,2 +1,2 @@
 bugfixes:
-- powershell - use the tmpdir set by `remote_tmp` for become/async tasks instead of the generic $env:TEMP - https://github.com/ansible/ansible/pull/40210
+- powershell - use the tmpdir set by ``remote_tmp`` for become/async tasks instead of the generic $env:TEMP - https://github.com/ansible/ansible/pull/40210

--- a/changelogs/fragments/vmware_remove_vm_sceanrio.yaml
+++ b/changelogs/fragments/vmware_remove_vm_sceanrio.yaml
@@ -1,3 +1,3 @@
 ---
 minor_changes:
-- Sceanrio guide for removing an existing virtual machine is added.
+- Scenario guide for removing an existing virtual machine is added.

--- a/changelogs/fragments/winrm-get_correct_user.yaml
+++ b/changelogs/fragments/winrm-get_correct_user.yaml
@@ -1,2 +1,2 @@
 bugfixes:
-- winrm - allow `ansible_user` or `ansible_winrm_user` to override `ansible_ssh_user` when both are defined in an inventory - https://github.com/ansible/ansible/issues/39844
+- winrm - allow ``ansible_user`` or ``ansible_winrm_user`` to override ``ansible_ssh_user`` when both are defined in an inventory - https://github.com/ansible/ansible/issues/39844

